scripts/report_errors.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+export ERROR_LOG="/tmp/error.log"
+
+function report_error_to_github {
+    if [ $? -eq 0 ]; then
+        exit 0
+    fi
+
+    if [ ! -z $CI ]; then
+        echo "posting error message to github"
+        post_error_to_github
+    fi
+}
+
+function post_error_to_github {
+    if [ -z $CIRCLE_PR_NUMBER ]; then
+        CIRCLE_PR_NUMBER="${CIRCLE_PULL_REQUEST//[^0-9]/}"
+    fi
+
+    GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/issues/$CIRCLE_PR_NUMBER/comments"
+
+    ESCAPED_ERROR_MSG=$(cat -e $ERROR_LOG | sed 's/"/\\\"/g' | sed 's/\\/\\\\/g')
```

Comment:
Shouldn't you first replace `\\` and then `"`? Otherwise the `\\` from the `"` is escaped again.

Reply:
Oh wow, never knew `cat` has options!

Reply:
Good catch. Didn't think about that. Will get that updated.